mlir/lib/Dialect/Rock/utility/AccelEmitter.cpp

```diff
@@ -843,7 +843,7 @@ AccelEmitterParams WmmaEmitter::initAccelEmitterParams(
   // and we want to do a(16x16) * b(16x16), 16 threads are loading a vector
   // of 16 Ks and the other 16 threads are replicating those values.
   int64_t waveSize = rock::lookupArchInfo(arch).waveSize;
-  if (arch.contains("gfx12")) {
+  if (!isGfx11) {
```

Comment:
Gfx10 would also fall in branch. is `gfx10` supported ?

Reply:
It doesn't have WMMA